openai-core/src/commonMain/kotlin/com.aallam.openai.api/run/RunRequest.kt

```diff
@@ -48,6 +51,45 @@ public data class RunRequest(
      * Keys can be a maximum of 64 characters long, and values can be a maximum of 512 characters long.
      */
     @SerialName("metadata") val metadata: Map<String, String>? = null,
+
+    /**
+     * What sampling temperature to use, between 0 and 2.
+     * Higher values like 0.8 will make the output more random, while lower values like 0.2 will make it more focused and deterministic.
+     */
+    @SerialName("temperature") val temperature: Int? = null,
+
+    /**
+     * An alternative to sampling with temperature, called nucleus sampling, where the model considers the results of the tokens with top_p probability mass.
+     * So 0.1 means only the tokens comprising the top 10% probability mass are considered.
+     */
+    @SerialName("top_p") val topP: Int? = null,
+
+    /**
+     * The maximum number of prompt tokens that may be used over the course of the run.
+     * The run will make a best effort to use only the number of prompt tokens specified, across multiple turns of the run.
+     * If the run exceeds the number of prompt tokens specified, the run will end with status `incomplete`.
+     * See `incomplete_details` for more info.
+     */
+    @SerialName("max_prompt_tokens") val maxPromptTokens: Int? = null,
+
+    /**
+     * The maximum number of completion tokens that may be used over the course of the run.
+     * The run will make a best effort to use only the number of completion tokens specified, across multiple turns of the run.
+     * If the run exceeds the number of completion tokens specified, the run will end with status `incomplete`.
+     * See `incomplete_details` for more info.
+     */
+    @SerialName("max_completion_tokens") val maxCompletionTokens: Int? = null,
```

Comment:
```suggestion
    @SerialName("max_completion_tokens") val maxCompletionTokens: Long? = null,
```
Same suggestion here